### PR TITLE
chore: generate command-not-found db instead of storing it as header files in git history

### LINKED ISSUE #13: Auto-update command database with cron jobs
It should be possible to do this. The cron job can be run once a week. This should also help us ensure that our repository is consistent with termux-packages git repo

### PR DESCRIPTION
**Pull request by thunder-coding**
From now onwards, there will be a new version of command-not-found only
when a new feature gets added to the program, or something gets changed.
Unlike earlier when new version of command-not-found was released when
the database was updated, now this won't happen. Instead just a revbump
of command-not-found will be done, so that users can get the new
commands.

On the packaging side, we just need to run term_setup_nodejs and run the
'generate-db.js' in source directory. In future a more reliant method
will be added so that other apt-packaged distributions can also use
command-not-found.

Closes #13

### CODE REVIEW

**Comment by licy183** [comment]
I think this is an excellent job! Now we could use Github Action and cron to auto-update `command-not-found`.
But I still have one suggestion which may be not important. CMake has `COMMAND` operation, and it could execute a local command during the building process. It seems more convenient if a `shell` script is used for generate headers rather than a `js` script, since we don't have to check the existence of `/usr/bin/node`.

**Comment by thunder-coding** [comment]
Thanks @licy183! I've applied changes as requested by you. 

If I get a review from @Grimler91, I'll pull this in, push a new tag, and update the version in our apt repos too.

File: generate-db.js
Comment (Grimler91) [inline generate-db.js:48]: Texlive's binaries are in `$PREFIX/bin/texlive/`, so rather than splitting on binPrefix.length it would be better to strip the dirname, or find last `/`. Right now texlive's programs are written as `texlive/pdflatex in the generated list

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Gave it a go and seem to work nicely! See comment about prefix length.

Readme could/should be updated as well, but I can do that after this is merged also.

File: generate-db.js
Comment (thunder-coding) [reply generate-db.js]: Thanks, fixed 👍

**Review by Grimler91 — APPROVED** [review]

**Pull request merged**
